RecoTracker/FinalTrackSelectors/python/earlyGeneralTracks_cfi.py

```diff
@@ -11,7 +11,8 @@
                       'detachedTripletStepTracks',
                       'mixedTripletStepTracks',
                       'pixelLessStepTracks',
-                      'tobTecStepTracks'
+                      'tobTecStepTracks',
+                      'siStripTripletStepTracks'
```

Comment:
I think we should add a modifier ruther than apply this change to the default tracking sequence

Reply:
Exactly what I suggested. Just a modifier that activates this iteration after tobtec. It can work whether with run2 or with run3. It just has to be added (when needed) in the cmsDriver commands after the Era. It won't be in the standard sequence. What do you think?